src/components/Tree/types.ts

```diff
@@ -18,8 +18,12 @@ export type TreeProps = {
     onExpand?: OnExpandCallback;
     onDrop?: OnDropCallback<{ id: string; sort: number }>;
     children: ReactNode;
+    /** Padding added to each TreeItem in the TreeContext. Default: {"top": 10, "right": 8, "bottom": 10, "left": 8} */
```

Comment:
```suggestion
    /** 
      * Padding added to each TreeItem in the TreeContext.
      * @default {"top": 10, "right": 8, "bottom": 10, "left": 8}
      */
```
(Indentation might be incorrect in the suggestion)
=> https://jsdoc.app/tags-default.html

Reply:
fixed